debian/changelog

```diff
@@ -1,3 +1,11 @@
+librtr0 (0.5.0.0) stable; urgency=high
```

Comment:
would it not be better to increment epoch with the next release?

Reply:
My understanding is that epoch bumps are the tool of last resort, to be used only when the upstream has screwed up the version numbering so badly that nothing else will work.  That does not seem to be the case here, so I didn't think it was necessary.

Reply:
We definitely should not increase epoch (that would be making it version number 1:0.5.0). I assume @mroethke  probably simply meant setting version to 0.5.1 with the next release.

Reply:
No, I meant epoch.